app/src/main/res/layout/topic_practice_fragment.xml

```diff
@@ -12,7 +12,7 @@
     android:layout_height="match_parent"
     android:clipToPadding="false"
     android:overScrollMode="never"
-    android:paddingBottom="172dp"
+    android:layout_marginTop="12dp"
```

Comment:
You can remove this margin top. As you are setting margin top 32 dp in topic_practice_header_view.

Reply:
No in header I am setting 24dp only...So adding 12dp to 24dp total makes 32dp. I have done in  that way.

Reply:
Ok got it. Make it consistent in both portrait and landscape.

Reply:
Done same as mockup